src/masternodes/mn_checks.h

```diff
@@ -249,6 +251,8 @@ struct CCreateMasterNodeMessage {
         if (!s.eof()) {
             READWRITE(timelock);
         }
+        READWRITE(rewardType);
```

Comment:
Would need a fork check. Not sure if we can extend this message since it has already been extended once before.

Reply:
Could do like last time I extended this custom TX message.

```
        if (!s.eof()) {
            READWRITE(rewardType);
        }
```